### PR TITLE
Feature newsweek replica

### PR DESCRIPTION
**Pull request by Dipeshtwis**
This project includes :

- Used bootstrap for many cases.
- uses media queries also for some cases like the font.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1039 @@
...
+            <div class="row magazine">
+               <div class="col-sm-12 col-md-3">
+                  <div class="card">
+                     <div class="image-card">
+                        <div class="caption">JUNE 05 ISSUE</div>
+                        <img class="card-img-top" src="assets/img/may-29-june-06-2020-cover.jpg" alt="Card image">
+                     </div>
+                     <a href="#">
+                        <div class="china-img mb-3">
+                           <h3 class="for-style for-hover">See all features ></h3>
```
Comment (BrittanyBlake) [inline index.html]: This should be an `<a>` tag as it is a link

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+h5,
+h6 {
+  font-weight: 600;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+h2 {
+  padding: 10px;
+  color: grey;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: You can use the bootstrap class `text-muted` to make this heading grey

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+
+header {
+  max-width: 1366px;
+  position: fixed;
+  top: 0;
+  z-index: 3;
+}
+
+article {
+  padding-top: 15px;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: You can use bootstrap classes for padding

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,559 @@
...
+  color: white;
+}
+
+.fas {
+  color: #dce0e6;
+}
+
+.navbar {
+  width: 100%;
+  background-color: white;
```
Comment (BrittanyBlake) [inline assets/css/style.css:121]: you can use bootstrap classes to make the background color white

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,559 @@
...
+  color: #f72310;
+}
+
+.form-search {
+  padding: 0 1rem;
+  position: relative;
+}
+
+.input-search {
+  border: none;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: you can use a bootstrap class to make the border none

File: assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+
+.image-card {
+  position: relative;
+}
+
+.caption {
+  position: absolute;
+  font: bold 17px/16px "Roboto Condensed", sans-serif;
+  color: #f72210;
+  text-transform: uppercase;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: you can use a bootstrap class to make the text uppercase

File: assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,559 @@
...
+  left: 80px;
+}
+
+.img-div-2 {
+  margin-top: -50px;
+  text-align: center;
+}
+
+.img-info {
+  text-align: center;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: you can use a bootstrap class to center the text

File: assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,559 @@
...
+
+  .bit {
+    height: 120px;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  h3 {
+    font-size: 15px;
+    font-weight: bold;
```
Comment (BrittanyBlake) [inline assets/css/style.css]: you can use a bootstrap class to set the font weight

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
****
###  Hi @Dipeshtwis 👋 You have done an amazing job so far!! 👍 However, the following issues need to be fixed before you can move forward 👩🏾‍💻  ⬇️ 
****
### Navigation bar:
- [x] Login and Subscribe should be links, not buttons. If clicking on the element will perform an action on the page, then the <button>tag should be used but if clicking on the element will navigate the user to a new page, such as a login page, then we should use the <a> tag. [You can check this article on semantic HTML for more details ](https://web.dev/use-semantic-html/). 
- [x] The subscribe button should be a bit have rounded corners like on the original page. (hint: you can use the bootstrap class rounded to achieve this)
- [x] The section containing the main navigation links should take up more of the width of the page. Please see the pictures below for more details:
The original page ⬇️ 
Screen Shot 2020-06-04 at 7 45 29 PM
Your clone ⬇️ 
Screen Shot 2020-06-04 at 7 45 46 PM

### In the Magazine:
- [x]  The section title (in the magazine) should have a bigger font size.
Your clone  ⬇️
Screen Shot 2020-06-04 at 7 55 00 PM
The original page ⬇️
Screen Shot 2020-06-04 at 7 55 06 PM
- [x] In the md screen size, the titles of the articles are not properly aligned. Please see screenshots below: 
The original page ⬇️
Screen Shot 2020-06-04 at 7 59 11 PM

Your clone  ⬇️ - "Cover" and "news" should be side-by-side as shown above ⬆️ 
Screen Shot 2020-06-04 at 7 59 18 PM

- [x]  In the sm screen size, there should be no gap between the magazine cover and the link to "see all features". Please check the screenshots below for more details:
The original page ⬇️
Screen Shot 2020-06-04 at 8 02 20 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 02 10 PM
- [x] In the sm screen, there should be no gap between the magazine cover and the second article photo:
The original page ⬇️
Screen Shot 2020-06-04 at 8 08 20 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 08 30 PM

### Featured Slideshow Section
- [x] The smaller articles on the side should take up the same height as the main articles. Please check the screenshots below for more details:
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 15 56 PM
The original page ⬇️
Screen Shot 2020-06-04 at 8 16 01 PM

### Subscribe & Newsletter Section
- [x] The subscribe and newsletter elements should be the same height.
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 21 04 PM
The original page ⬇️
Screen Shot 2020-06-04 at 8 21 15 PM


- [x] The font sizes, font-weight, and font color in your clone are not the same as the original in both the newsletter and subscribe elements.
- [x] The subscribe button should have rounded corners. Please compare with the original page:
The original page ⬇️
Screen Shot 2020-06-04 at 8 24 26 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 24 32 PM
- [x] The text in the subscribe section overlaps with the image as you move to a smaller screen. (1024 px)
Screen Shot 2020-06-04 at 8 27 37 PM

- [x] In the md screen, there should be more space between the text and the images in both the subscribe and the newsletter elements.
The original page ⬇️
Screen Shot 2020-06-04 at 8 29 24 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 29 32 PM

### Footer
- [x] The Newsweek logo in the footer should be larger.
- [x] The footer links should be separated by a line ( | ). Please compare with the original page:
The original page ⬇️
Screen Shot 2020-06-04 at 8 35 42 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 35 47 PM

- [x] On the sm screen, the Newsweek logo, the copyright statement, and the social media icons should be properly aligned. Please see below:
The original page ⬇️
Screen Shot 2020-06-04 at 8 37 12 PM
Your clone  ⬇️
Screen Shot 2020-06-04 at 8 37 18 PM

### General
- [x] Please check your CSS file and ensure you are using CSS only when absolutely necessary. I have added some comments of examples where bootstrap should be used in place of CSS. However, my comments do not indicate the only places you can use bootstrap so please double check your css.  You may find this [bootstrap cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) useful.


### Please make the required changes and resubmit for a new code review once you are ready 🙏 

Happy Coding! 😄 


P.S.
If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake).

File: assets/css/style.css (hunk 3 above)
Comment (Dipeshtwis) [reply assets/css/style.css]: Now I used

**Comment by Dipeshtwis** [comment]
The updated project include : 
 - used margin, padding, border, and many things with the help of bootstrap.
- similar styling to all the elements as Newsweek website.

Thank you for review this project in advance and also for your time and energy.

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT :computer:  (TSE)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1050 @@
...
+                        <form class="form-search">
+                           <input type="text" class="input-search border-0" name="search" placeholder="Search">
+                           <i class="fas fa-search fa-lg"></i>
+                        </form>
+                     </li>
+                  </ul>
+               </div>
+            </nav>
+         </header>
+         <div class="section">
```
Comment (peterrobert) [inline index.html:84]: Kindly insert all your main section inside the main tag. For more information about the main tag kindly check [here](https://www.w3schools.com/TAgs/tag_main.asp#:~:text=Definition%20and%20Usage,site%20logos%2C%20and%20search%20forms.)

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
**Great work @Dipeshtwis :clap:  However there are still issues that are preventing me from approving the project. Kindly work on the following**

- [x] Kindly double check the suggestions made by the previous TSE. I have noticed that you haven't made some changes as required. For example, the menu for the desktop view should fill the entire container.
menu_problem

- [x] At the mobile view your site is losing responsiveness. Kindly improve on this
menu-problem

Very good progress :clap:  Just fix those issues and resubmit your work. 

Happy coding :computer: 

**just reach out to me in case you have any questions. @peterrobert via slack **

**Comment by Dipeshtwis** [comment]
The project updates include :

- link instead of a button to login and subscribe.
- Menu filling the maximum width to the container.
- Made responsive

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

Good job so far! There is a thing you need to do in order to achieve approval.

The **Newsletter section** is not being properly displayed.

**Original**

Screen Shot 2020-06-05 at 8 36 36 AM

**Yours**

Screen Shot 2020-06-05 at 8 36 44 AM

Please fix that, and once you're done submit another CRR.

You can find me in slack as **srodrig**, happy coding! :wave:

**Comment by Dipeshtwis** [comment]
The update project include

- made newsletter-text to align in the center

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi, @Dipeshtwis, you did awesomely!!! :+1:  Having fixed the alignment in the subscribe section :100: I, therefore, approve the project :fireworks: Go ahead and merge to master, submit the project completion form.

Good luck with your next submissions :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**